src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/GregtechMetaTileEntity_IndustrialMixer.java

```diff
@@ -46,6 +50,18 @@ public GregtechMetaTileEntity_IndustrialMixer(final String aName) {
         CASING_TEXTURE_ID = TAE.getIndexFromPage(2, 2);
     }
 
+    @Override
+    public void saveNBTData(NBTTagCompound aNBT) {
+        super.saveNBTData(aNBT);
+        aNBT.setBoolean("isBussesSeparate", isBussesSeparate);
+    }
+
+    @Override
+    public void loadNBTData(final NBTTagCompound aNBT) {
+        super.loadNBTData(aNBT);
+        isBussesSeparate = aNBT.getBoolean("isBussesSeparate");
```

Comment:
is this a deliberate choice to make all past instance of mixer to default to not separate bus?

Reply:
I'm currently not aware of any method to give the NBT a fallback. If you can tell me how its done I can make a 2nd PR.

Reply:
Check if there's existing tag with `aNBT.hasKey`